src/calculateRentalCost.js

```diff
@@ -28,7 +28,16 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let count = 0;
```

Comment:
Variable name should be meaningful:
```suggestion
  let discount = 0;
```